### ISSUE TITLE
230718/tensorflow_회귀,분류 실습

### ISSUE THREAD

**Issue by doxgxxn** [doxgxxn/WILT#2]
def get_model_lr(initial_lr=0.01):
            model = keras.Sequential()
            model.add(layers.Flatten(input_shape=(28,28)))
            model.add(layers.Dense(256, activation='relu'))
            model.add(layers.Dense(128, activation='relu'))
            model.add(layers.Dense(64, activation='relu'))
            model.add(layers.Dense(10, activation='softmax', name='output_layer'))
            
            lr_sch = optimizers.schedules.ExponentialDecay(initial_learning_rate=initial_lr, 
                                                           decay_steps=len(train_dataset) * 5, 
                                                           decay_rate=0.5, 
                                                           staircase=True)
            
            model.compile(optimizer=optimizers.Adam(learning_rate=lr_sch),
                          loss='categorical_crossentropy', metrics=['accuracy'])
            
            return model
        
        
        
        
        -----------------
        
        lr_callback = keras.callbacks.ReduceLROnPlateau(monitor='val_loss', 
                                                        factor=0.5, 
                                                        patience=3, 
                                                        verbose=2
                                                       )
        
        hist = model_lr2.fit(train_dataset, epochs=N_EPOCH+10, validation_data=test_dataset, 
                            callbacks=[lr_callback])
        
        
        
        
        
        
        optimizers.schedules.ExpotentialDecay(inital_learning_rate=x, Decay_rate, Decay_steps, staircase=True)
        
        keras.Callbacks.ReducedLROnplateau(moniter='val_loss', factor=0.5, patience=3, verbose=2)
